api/src/main/java/marquez/db/RunDao.java

```diff
@@ -437,4 +438,35 @@ WITH RECURSIVE job_names AS (
       )
       """)
   Optional<Run> findByLatestJob(String namespace, String jobName);
+
+  /**
+   * @param runUuid
+   */
+  @SqlQuery(
+      """
+      SELECT
+        run_uuid,
+        JSON_AGG(facet ORDER BY lineage_event_time) AS facets
+      FROM
+      run_facets_view
```

Comment:
Shouldn't this be a part of `RunFacetsDao` ?

Reply:
Done.